cmd/dynatrace/logsCmd.go

```diff
@@ -100,13 +111,13 @@ func NewCmdLogs() *cobra.Command {
 	return logsCmd
 }
 
-func GetLinkToWebConsole(dtURL string, since int, finalQuery string) (string, error) {
+func GetLinkToWebConsole(dtURL string, from interface{}, to interface{}, finalQuery string) (string, error) {
```

Comment:
Can we also use string here instead? Just looking at the signature of this function, it is not clear that it only takes valid Dynatrace from and to values, mainly now() or some variation of that in our code. Having interface here might lead to passing in timestamps in time.Time format